recipes/osqp-eigen/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+mkdir build && cd build
+
+cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
```

Comment:
```suggestion
cmake ${CMAKE_ARGS} \
```
See https://github.com/conda-forge/staged-recipes/pull/15301#discussion_r652075460 and https://github.com/conda-forge/staged-recipes/pull/15140#discussion_r646172798 .